src/scenes/NftDetailPage/NftDetailNote.tsx

```diff
@@ -0,0 +1,241 @@
+import { BodyRegular } from 'components/core/Text/Text';
+import Spacer from 'components/core/Spacer/Spacer';
+import TextButton from 'components/core/Button/TextButton';
+import { useCallback, useState, useMemo, useRef } from 'react';
+import { TextAreaWithCharCount } from 'components/core/TextArea/TextArea';
+import unescape from 'lodash.unescape';
+import styled from 'styled-components';
+import useUpdateNft from 'hooks/api/nfts/useUpdateNft';
+import Markdown from 'components/core/Markdown/Markdown';
+import breakpoints from 'components/core/breakpoints';
+import ErrorText from 'components/core/Text/ErrorText';
+import formatError from 'errors/formatError';
+import { GLOBAL_FOOTER_HEIGHT } from 'components/core/Page/constants';
+
+const MAX_CHAR_COUNT = 400;
+const MIN_NOTE_HEIGHT = 150;
+
+type Props = {
+  nftCollectorsNote?: string;
+  nftId: string;
+  userOwnsAsset: boolean;
+};
+
+function NftDetailNote({ nftCollectorsNote, nftId, userOwnsAsset }: Props) {
+  // Generic error that doesn't belong to collector's note
+  const [generalError, setGeneralError] = useState('');
+
+  const [noteHeight, setNoteHeight] = useState(MIN_NOTE_HEIGHT);
+  const [isEditing, setIsEditing] = useState(false);
+
+  const [collectorsNote, setCollectorsNote] = useState(nftCollectorsNote ?? '');
+  const unescapedCollectorsNote = useMemo(() => unescape(collectorsNote), [collectorsNote]);
+
+  const hasCollectorsNote = useMemo(() => collectorsNote.length > 0, [collectorsNote]);
+
+  const collectorsNoteRef = useRef<HTMLDivElement>(null);
+
+  const scrollDown = () => {
+    if (collectorsNoteRef.current) {
+      collectorsNoteRef.current.scrollIntoView({
+        block: 'end',
+        inline: 'nearest',
+        behavior: 'smooth',
+      });
+    }
+  };
+
+  const handleEditCollectorsNote = useCallback(() => {
+    setIsEditing(true);
+
+    // TODO Expand note to full height first time it is opened
+    // Currently user has to type first to expand it fully
```

Comment:
is this still needed?

Reply:
Yes, this is only relevant when the user reloads the page or visits an existing long note and tries to edit it. (I can clarify in the TODO comment as well.) There's a video of the problem this in the edited PR under "Remaining issues"